src/component/App.jsx

```diff
@@ -88,11 +90,14 @@ function App() {
                     social
                   </Route>
                   <Route exact path="/settings">
-                    add ingredient
+                    <SettingsStart />
                   </Route>
                   <Route exact path="/social/:reviewId">
                     social/reviewId
                   </Route>
+                  <Route path="/profile">
```

Comment:
any specific reason why you're just using path here instead of exact path? I know it's not going to make a difference since there's no subpath after profile, but I just ask for consistency, since you've been using exact path throughout.

Reply:
I probably should update the App.jsx, just did an exact path for now primarily because I wasn't sure if the other components will have sub paths.  Since I knew for the /profile it won't have any sub paths, I didn't add the exact. Moving forward, should I change this right now to an exact path, or wait until we're close to production and refactor the exact keyword as needed?